### DISCUSSION TITLE
I can not veiw on two sreens at the seime time

### DISCUSSION THREAD

**Discussion by sergio4567** [utmapp/UTM#6802] [UTM Help (macOS)]
**Describe the issue**  
Good morning, I have a MacBook and I have UTM installed. It works very well, but I have an external screen connected with a cable and I cannot have UTM Windows on both screens at the same time. I think the problem is that it only detects one screen. I need it for work and I would like someone to help me. Thank you very much.

**Configuration**  
* UTM Version: QEMU 7.2 ARM virtual Machine (alias of virt-7.2) (virt)
* macOS Version: Sequoia 15.1
* Mac Chip (Intel, M1, ...): M2

**Comment by osy** [comment]
You need to emulate a second display. In the VM settings, add a new display device.
